corehq/apps/sso/views/accounting_admin.py

```diff
@@ -68,7 +69,7 @@ def rows(self):
         def _idp_to_row(idp):
             edit_url = reverse(EditIdentityProviderAdminView.urlname, args=(idp.id,))
             return [
-                f'<a href="{edit_url}">{idp.name}</a>',
+                format_html('<a href="{}">{}</a>', edit_url, idp.name),
```

Comment:
I think the way its written here is fine, but just pointing out if you wanted to mimic the previous version (with named parameters), `format_html()` also supports that syntax: 
`format_html('<a href="{edit_url}">{name}</a>', edit_url=edit_url, name=idp.name)`